templates/github/build-release-policy.tmpl

```diff
@@ -0,0 +1,81 @@
+name: build-release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release_policy:
+    runs-on: ubuntu-latest
+    name: build
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read config
+      id: config
+      uses: opcr-io/ga-yaml-parser@v0.1.1
```

Comment:
maybe we should be a little less specific for versions, so we can deliver fixes more easily?
```suggestion
      uses: opcr-io/ga-yaml-parser@v0.1
```